build.gradle

```diff
@@ -221,6 +221,7 @@ configurations.all {
         force "net.bytebuddy:byte-buddy-agent:1.14.6"
         force "com.google.code.gson:gson:2.8.9"
         force "junit:junit:4.13.2"
+        force "org.apache.commons:commons-lang3:3.12.0"
```

Comment:
Let's the read the version from core like `${versions.commonslang}` and not force it.

Reply:
I meant let's change the line 142 to 
` implementation group: 'org.apache.commons', name: 'commons-lang3', version: '${versions.commonslang}`, we don't need this line then at all.

Reply:
yes, changed it on line 142. Didn't realize we're defining a version here